commands/issue/create/issue_create.go

```diff
@@ -134,6 +136,8 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 	issueCreateCmd.Flags().BoolVar(&opts.Web, "web", false, "continue issue creation with web interface")
 	issueCreateCmd.Flags().IntSliceVarP(&opts.LinkedIssues, "linked-issues", "", []int{}, "The IIDs of issues that this issue links to")
 	issueCreateCmd.Flags().StringVarP(&opts.IssueLinkType, "link-type", "", "relates_to", "Type for the issue link")
+	issueCreateCmd.Flags().StringVarP(&opts.TimeEstimate, "estimate", "e", "", "Set time estimate for the issue")
+	issueCreateCmd.Flags().StringVarP(&opts.TimeSpent, "spend", "s", "", "Set time spent for the issue")
```

Comment:
I changed these to `time-spent`and `time-estimate`.

This should be descriptive enough so it can't be confused without anything else, while being discoverable and easy to understand.